src/ol/format/gml/gmlbaseformat.js

```diff
@@ -99,6 +106,11 @@ ol.format.GMLBase.prototype.readFeatures_ = function(node, objectStack) {
     }
     var parsers = {};
     var parsersNS = {};
+    if (goog.object.get(context, 'useArrayPusher') === true) {
+      goog.object.set(this.FEATURE_COLLECTION_PARSERS[ol.format.GMLBase.GMLNS],
+          'featureMember',
+          ol.xml.makeArrayPusher(ol.format.GMLBase.prototype.readFeatures_));
```

Comment:
> I am not changing it on the prototype but on the instance (this.FEATURE_COLLECTION_PARSERS)

I think `this.FEATURE_COLLECTION_PARSERS` refers to the object [defined](https://github.com/bartvde/ol3/blob/ogr-gml2/src/ol/format/gml/gmlbaseformat.js#L130-L137) on the prototype.